regression/cbmc-library/posix_memalign-01/test.desc

```diff
@@ -1,8 +1,8 @@
 CORE
 main.c
---pointer-check --bounds-check
+--pointer-check --bounds-check --malloc-may-fail --malloc-fail-null
```

Comment:
I added the flags, so now the test considers malloc can fail. However, this doesn't change the expected output of this test, since it's resilient to a failed malloc. @hannes-steffenhagen-diffblue

Reply:
@feliperodri I'd recommend to reverse the changes here. Tests should be self contained and not contain flags that aren’t relevant for the test. This test is unrelated to `--malloc-may-fail` and hence shouldn’t use the flag.

Reply:
@hannes-steffenhagen-diffblue I'll send a new PR that update these flags. I'll keep the failed regression test with the malloc flags and I'll remove them from the this test case.